### ISSUE TITLE
Fighting the Gaia Guardians

### ISSUE THREAD

**Issue by 61352151511** [Vazkii/Botania#1076]
So I know the beacon raising you if you try to fly is probably an intended feature, however once the beacon gets me to a certain height the guardian despawns, and there's no way to get out of the beacon that I know of. So I put up with the first one despawning and then fought a second and killed it, I made the gaia spirit ingot and went to fight the tier II guardian, I didn't even fly this time I just jumped and suddenly started getting sucked up into the air. I do have a lot of buffs that make me jump higher but it bugged me that simply jumping gets me sucked into the beacon and the guardian despawned. Then with the last 4 gaia spirits I had from the first gaia, I made a second ingot, didn't jump or fly that time knowing what it would get me into, one of those pixies hit me and somehow into the beacon I go. I know the fight is supposed to be difficult, but it shouldn't be impossible to get down from beam that despawns guardian, wasting your resources difficult.

Basically the player getting moved up for jumping or flying eventually gets past the 32 block radius and the guardian despawns itself which is really annoying

**Comment by Lazersmoke** [comment]
`the beacon raising you if you try to fly is probably an intended feature` is not intended AFAIK, and has never happened to me...

**Comment by CatDany** [comment]
@Lazersmoke Yep, I can confirm, haven't seen this before

**Comment by TheWhiteWolves** [comment]
Well I don't think that's supposed to happen, as far as I remember it's only supposed to cancel any flight abilities.

**Comment by TheWhiteWolves** [comment]
Also what version are you running against and have you tried it with only botania installed.

**Comment by 61352151511** [comment]
I just tried it with just botania (191) and extra utilities (That's the method I was using to fly) and it didn't happen. I'm not really sure what could be causing it as when it happened it was with 130 mods, but maybe as a temporary workaround until I figure out what mod causes it could the area of the guardian despawning be changed from X, Y, Z dependent to just X, Z dependent and the Y axis wouldn't matter?

**Comment by TheWhiteWolves** [comment]
Probably won't bother as it's a minor issue but I might take a look at it some time, for now I'm going to close this issue, you can still comment on it when you find what mod causes the issue.

**Comment by TheWhiteWolves** [comment]
Probably won't bother as it's a minor issue but I might take a look at it some time, for now I'm going to close this issue, you can still comment on it when you find what mod causes the issue.
